### PR TITLE
Add hadoop cluster

### PR DESCRIPTION
**Pull request by 251744647**
Add hadoop cluster on ubuntu. 1 name node, 2 data nodes by default. User can define the data nodes number. Also enable zabbix monitoring on all nodes.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@251744647__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

File: hadoop-cluster-ubuntu/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+    "_artifactsChinaLocationSasToken": {
+      "type": "securestring",
+      "metadata": {
+      "description": "The sasToken required to access _artifactsLocation.  When the template is deployed using the accompanying scripts, a sasToken will be automatically generated."
+      },
+      "defaultValue": ""
+    }
+  },
+  "variables": {
+    "environmentAzureCloud": {
```
Comment (bmoore-msft) [inline hadoop-cluster-ubuntu/azuredeploy.json]: We shouldn't have different vars for different clouds but be using the reference function to get the desired endpoints...  We need to make this work with one set of _artifacts* variables

Comment (251744647) [inline hadoop-cluster-ubuntu/azuredeploy.json]: @bmoore-msft  Updated.

File: hadoop-cluster-ubuntu/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,289 @@
...
+      "serviceEndPoint": ".blob.core.windows.net/vhds/",
```
Comment (bmoore-msft) [inline hadoop-cluster-ubuntu/azuredeploy.json]: See #9 in the BP guide for handling uri's across clouds

Comment (251744647) [inline hadoop-cluster-ubuntu/azuredeploy.json]: @bmoore-msft  No multiple clouds, just a single cloud. Updated.

**Comment by bmoore-msft** [comment]
Closing old PR

**Pull request closed without merging**